### ISSUE TITLE
Error when installing plugin -- @oclif/config@1.18.17: Package no longer supported.

### ISSUE THREAD

**Issue by VincenzoMeschi** [forcedotcom/salesforcedx-vscode#5591]
### Summary

I am attempting to install this plugin. I am connected to a Default org and am all setup. 

I am getting the error 
`@salesforce/lwc-dev-server > @oclif/config@1.18.17: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.`  
when attempting to run the base 
`sf plugins install @salesforce/lwc-dev-server` 
command. 

### Additional information

```bash
PS D:\Projects\SF_HelpCenter> sf plugins install @salesforce/lwc-dev-server
Polling for new version(s) to become available on npm... done
Successfully validated digital signature for @salesforce/lwc-dev-server.
Finished digital signature check.
Installing plugin @salesforce/lwc-dev-server@latest... failed
 »   Warning: @salesforce/lwc-dev-server > @oclif/config@1.18.17: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
    Error: C:\Program Files\sf\client\node_modules\yarn\bin\yarn.js add @salesforce/lwc-dev-server@latest --non-interactive --mutex=file:C:\Users\me\AppData\Local\sf\yarn.lock --preferred-cache-folder=C:\Users\me\AppData\Local\sf\yarn --check-files exited with code 1
```

**Salesforce Extension Version in VS Code**:
`Salesforce CLI Integration v60.11.0`

**Salesforce CLI Version**:
`@salesforce/cli/2.24.4 win32-x64 node-v20.10.0`

**OS and version**:
Windows 10 Home

**VS Code version**:
1.87.2

**Comment by peternhale** [comment]
@VincenzoMeschi The error you are seeing is not due to @oclif/config deprecation but the error that follows it. Would you provide the full terminal output from your attempt to install lwc dev server?

**Comment by VincenzoMeschi** [comment]
This was all that I got after attempting to install.

```bash
PS D:\Projects\SF_HelpCenter> sf plugins install @salesforce/lwc-dev-server
Polling for new version(s) to become available on npm... done
Successfully validated digital signature for @salesforce/lwc-dev-server.
Finished digital signature check.
Installing plugin @salesforce/lwc-dev-server@latest... failed
 »   Warning: @salesforce/lwc-dev-server > @oclif/config@1.18.17: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
    Error: C:\Program Files\sf\client\node_modules\yarn\bin\yarn.js add @salesforce/lwc-dev-server@latest --non-interactive --mutex=file:C:\Users\me\AppData\Local\sf\yarn.lock --preferred-cache-folder=C:\Users\me\AppData\Local\sf\yarn --check-files exited with code 1
```

**Comment by peternhale** [comment]
@VincenzoMeschi Try this `sf plugins install @salesforce/lwc-dev-server --dev-debug`

**Comment by ajmAAid** [comment]
@peternhale I get the exact same error as @VincenzoMeschi and can provide the full terminal output. See below. My apologies in advance if this is not good protocol to comment in this way, this is my first time commenting on a open source repo and looking at README and CONTIRBUTING I can't see any particular protocols related to comments.

```Installing plugin @salesforce/lwc-dev-server@latest... failed
Warning: @salesforce/lwc-dev-server > @oclif/config@1.18.17: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
    at Object.warn (C:\Program Files\sf\client\node_modules\@oclif\core\lib\errors\index.js:75:15)
    at YarnMessagesCache.flush (file:///C:/Program%20Files/sf/client/node_modules/@oclif/plugin-plugins/lib/util.js:120:20)
    at PluginsInstall.run (file:///C:/Program%20Files/sf/client/node_modules/@oclif/plugin-plugins/lib/commands/plugins/install.js:142:49)
    at async PluginsInstall._run (C:\Program Files\sf\client\node_modules\@oclif\core\lib\command.js:304:22)
    at async Config.runCommand (C:\Program Files\sf\client\node_modules\@oclif\core\lib\config\config.js:417:25)
    at async run (C:\Program Files\sf\client\node_modules\@oclif\core\lib\main.js:85:16)
  oclif-perf Process Uptime: 22285.8654ms +0ms
  oclif-perf Oclif Time: 21773.4863ms +1ms
  oclif-perf Init Time: 146.7077ms +0ms
  oclif-perf Config Load Time: 160.3756ms +0ms
  oclif-perf   • Root Plugin Load Time: 18.5204ms +1ms
  oclif-perf   • Plugins Load Time: 122.6841ms +0ms
  oclif-perf   • Commands Load Time: 14.4086ms +0ms
  oclif-perf Core Plugin Load Time: 116.0162ms +1ms
  oclif-perf User Plugin Load Time: 5.4918ms +0ms
  oclif-perf Linked Plugin Load Time: 0.0000ms +0ms
  oclif-perf Plugin Load Times: +0ms
  oclif-perf   @oclif/plugin-autocomplete: 102.7491ms +1ms
  oclif-perf   @oclif/plugin-commands: 99.4704ms +0ms
  oclif-perf   @oclif/plugin-help: 94.3441ms +0ms
  oclif-perf   @oclif/plugin-not-found: 88.5115ms +0ms
  oclif-perf   @oclif/plugin-update: 85.5979ms +1ms
  oclif-perf   @oclif/plugin-plugins: 84.9676ms +0ms
  oclif-perf   @oclif/plugin-search: 81.9221ms +0ms
  oclif-perf   @oclif/plugin-which: 73.3021ms +0ms
  oclif-perf   @oclif/plugin-version: 72.0974ms +1ms
  oclif-perf   @oclif/plugin-warn-if-update-available: 68.8468ms +0ms
  oclif-perf   @salesforce/plugin-apex: 62.2010ms +0ms
  oclif-perf   @salesforce/plugin-auth: 57.3603ms +1ms
  oclif-perf   @salesforce/plugin-data: 57.0189ms +0ms
  oclif-perf   @salesforce/plugin-deploy-retrieve: 56.4747ms +0ms
  oclif-perf   @salesforce/plugin-info: 52.5213ms +0ms
  oclif-perf   @salesforce/plugin-limits: 51.4564ms +1ms
  oclif-perf   @salesforce/plugin-marketplace: 49.3830ms +0ms
  oclif-perf   @salesforce/plugin-org: 48.8339ms +0ms
  oclif-perf   @salesforce/plugin-packaging: 45.2034ms +0ms
  oclif-perf   @salesforce/plugin-schema: 43.1796ms +1ms
  oclif-perf   @salesforce/plugin-settings: 41.9809ms +0ms
  oclif-perf   @salesforce/plugin-source: 40.7503ms +0ms
  oclif-perf   @salesforce/plugin-sobject: 40.0530ms +0ms
  oclif-perf   @salesforce/plugin-telemetry: 37.5856ms +0ms
  oclif-perf   @salesforce/plugin-templates: 35.4978ms +0ms
  oclif-perf   @salesforce/plugin-trust: 32.0449ms +1ms
  oclif-perf   @salesforce/plugin-user: 30.7445ms +0ms
  oclif-perf   root: 18.5204ms +0ms
  oclif-perf   lightning-flow-scanner: 3.9999ms +0ms
  oclif-perf Hook Run Times: +1ms
  oclif-perf   init: +0ms
  oclif-perf     total: 144.0021ms +1ms
  oclif-perf     @oclif/plugin-warn-if-update-available(./lib/hooks/init/check-update): 142.7798ms +0ms
  oclif-perf     @salesforce/plugin-settings(./lib/hooks/init/load_config_meta): 136.5937ms +0ms
  oclif-perf     @oclif/plugin-update(./dist/hooks/init.js): 44.3599ms +0ms
  oclif-perf   prerun: +1ms
  oclif-perf     total: 777.4894ms +0ms
  oclif-perf     @salesforce/cli(./dist/hooks/prerun): 13.4604ms +0ms
  oclif-perf     @salesforce/plugin-telemetry(./lib/hooks/telemetryPrerun.js): 773.4077ms +0ms
  oclif-perf   plugins:preinstall: +0ms
  oclif-perf     total: 4189.8797ms +1ms
  oclif-perf     @salesforce/cli(./dist/hooks/pluginsPreinstall.js): 4189.3546ms +0ms
  oclif-perf   plugins:preinstall:verify:signature: +0ms
  oclif-perf     total: 4185.2045ms +1ms
  oclif-perf     @salesforce/plugin-trust(./lib/hooks/verifyInstallSignature.js): 4182.8135ms +0ms
  oclif-perf Command Load Time: 0.0000ms +0ms
  oclif-perf Command Run Time: 21627.0043ms +0ms
Error: C:\Program Files\sf\client\node_modules\yarn\bin\yarn.js add @salesforce/lwc-dev-server@latest --non-interactive --mutex=file:C:\Users\ajm\AppData\Local\sf\yarn.lock --preferred-cache-folder=C:\Users\ajm\AppData\Local\sf\yarn --check-files exited with code 1
    at ChildProcess.<anonymous> (file:///C:/Program%20Files/sf/client/node_modules/@oclif/plugin-plugins/lib/yarn.js:117:28)
    at ChildProcess.emit (node:events:514:28)
    at ChildProcess._handle.onexit (node:internal/child_process:294:12)
  sf:telemetry Spawning "C:\Program Files\sf\client\bin\node.exe C:\Program Files\sf\client\node_modules\@salesforce\plugin-telemetry\processes\upload.js C:\Users\ajm\AppData\Local\sf C:\Users\ajm\AppData\Local\Temp\sf-telemetry\telemetry-85e126340ecd39ba5ba4a952ac1f76444b958409.log" +21s```

**Comment by daphne-sfdc** [comment]
Hi @VincenzoMeschi and @ajmAAid,

The LWC dev server is now deprecated, so we will not be actively maintaining this plugin.  However, you can still continue to use it as is.

As for the installation issue, I'm able to install the `@salesforce/lwc-dev-server` plugin when I'm using a CLI version that runs on Node 18 but not a CLI version that runs on Node 20.  (Run `sf --version` to check - the last part tells you what Node version your CLI is running on.)

If you installed the CLI using the installer, it comes bundled with Node 20, as the CLI Team has already done the Node upgrade a few months ago.  In that case, you'll need to uninstall your existing CLI version and install a new one via npm.  (Uninstalling the existing version is important - otherwise your computer and VSCode are going to get confused about which one to use!)

Here are the instructions to install the new CLI on Node 18 via npm:
1. Uninstall the old version by following these instructions: https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_uninstall.htm
2. Ensure the CLI is uninstalled by making sure `sfdx --version` and `sf --version` don't result in anything
3. Install Node 18 (`nvm install 18` if you have nvm; otherwise https://nodejs.org/en/download/current and select Node 18 from the dropdown)
4. Run `npm install @salesforce/cli --global` in the Terminal to install the latest version of the Salesforce CLI
5. `sf plugins install @salesforce/lwc-dev-server`

Let me know if this fixes your installation issue!
